src/src/components/navigation/NavigationBar.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="navigation-bar ">
+    <div class="navigation-content">
+      <div class="navigation-logo">
+        <fa icon="fingerprint" class="navigation-icon"></fa>
+      </div>
+      <h2>{{ siteTitle }}</h2>
+      <NavigationMenu :items="navigationItems" />
+    </div>
+    <Profile/>
+    <Notification />
+    <Dialog />
+  </div>
+</template>
+
+<script>
+import { computed, defineComponent } from 'vue';
+import store from '../../store/index';
+import NavigationMenu from './NavigationMenu.vue';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
import NavigationMenu from "./NavigationMenu.vue";
```